CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [1.19.0](https://github.com/aws-observability/aws-rum-web/compare/v1.18.0...v1.19.0) (2024-07-09)
+
+
+### Features
+
+* support unique credential cookie names ([#560](https://github.com/aws-observability/aws-rum-web/issues/560)) ([a440016](https://github.com/aws-observability/aws-rum-web/commit/a440016e402ed03e5fb463e13d1bdde13dfe5f40))
```

Comment:
nit: We've historically capitalized the first letter in the commit message. E.g., `feat: Support unique credential cookie names`.

This was, in part, so that entries in the CHANGELOG would be capitalized. Can we use this convention more consistently going forward?